### ISSUE TITLE
Export/Import encrypted notes

### ISSUE THREAD

**Issue by dsamersoff** [martinstoeckli/SilentNotes#225]
It would be nice to be able to export notes from the safe in encrypted form and read them back. This allows the user to keep their own encrypted backup or share notes with another SilenNotes instance, but does not enable cloud sync.

PS: Thank you for an excellent program.

**Comment by martinstoeckli** [comment]
Yes this is not the first time somebody asked for a local backup and it is somewhat related to https://github.com/martinstoeckli/SilentNotes/issues/96 . It would be a good feature to have and nothing speaks against an implementation, there are just other features with higher priority.

One thing that would have to be solved is, when a user has a synchronization in place and wants to restore a backup, then the sync would immediately overwrite the loaded backup (the notes of the backup will usually be older).

In the meantime one can setup a local WebDav instance in the private network, e.g. with a local NextCloud/OwnCloud instance. This has even the advantage that multiple devices can synchronize the notes. Thank you for writing the suggestion.

**Comment by martinstoeckli** [comment]
In version 8.0.4 I implemented a hidden feature, which allows to backup the repository to a directory of the locale device and to recover this backup. It is not yet an official feature because it replaces the current repository and therefore destroys all its notes.

Nevertheless, you can test it out, on the "Information" dialog, click 5 times on the logo and the page will expand with more information and the backup/recovery buttons.
